api/tests/test_winery.py

```diff
@@ -60,6 +60,27 @@ def test_winery_endpoint_create_should_not_be_allowed(self):
         )
         self.assertEqual(status.HTTP_401_UNAUTHORIZED, response.status_code)
 
+    def test_winery_approval(self):
+        admin = WineUser.objects.create_superuser(
+            email='admin@admin.com',
+            password='12345678',
+            first_name='User',
+            last_name='Test',
+            gender=GENDER_OTHER,
+            language=LANGUAGE_ENGLISH,
+            country=Country.objects.create(name='Test'),
+        )
+        winery = Winery.objects.create(name='Test Winery')
+        self.assertEqual(winery.available_since, None)
```

Comment:
assertIsNone and assertIsNotNone are more idiomatic :)

Reply:
Resolved